### ISSUE TITLE
Stripping namespace prefixes in JSON 

### ISSUE THREAD

**Issue by Pudgie** [brunato/xmlschema#113]
Is there a way to remove namespace prefix from a converted json from xml.
For example xml:
```
<reports ns="http://www.w3.org/2005/Atom" xmlns:georss="http://www.georss.org/georss">
       <georss:point>40.7111 -73.9565</georss:point>
<reports>
```

Desired JSON from xmlschema.to_json or json.dumps(xmlschema.to_dict):
```
{
   reports: { 
            "point":"40.7111 -73.9565"
       }
}
```
instead of "georss:point":"40.7111 -73.9565".

Thanks!

**Comment by brunato** [comment]
Currently there is no way to remove the namespace prefixes if there are multiple namespaces. The only method now is to work with elements of one namespace.

Providing the option `process_namespaces=True` causes  to leave the format `{namespace}name` to names, that is more complicated to use than prefixed format.
 
Anyway, let me thinking if an option to strip namespaces info during data conversion has sense or not ...

**Comment by brunato** [comment]
Stripping all prefixes may has sense for some others and is not complicated to do this ... I'll add an option *strip_namespaces* into the next release.

**Comment by brunato** [comment]
An option for stripping all namespace information during the decoding operation is available with new release v1.0.13.
Bye
